openshift-knative-operator/cmd/operator/kodata/ingress/0.24/3-500-controller.yaml

```diff
@@ -15,15 +15,15 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: networking-istio
+  name: net-istio-controller
```

Comment:
@nak3 before I forget: We probably need to do a cleanup pass here because I don't think our version of the operator automatically cleans this up.